src/types.ts

```diff
@@ -154,6 +154,7 @@ export interface HomeAssistant {
 
   vibrate: boolean;
   dockedSidebar: "docked" | "always_hidden" | "auto";
+  defaultPanel?: string;
```

Comment:
Let's always set it.
```suggestion
  defaultPanel: string;
```